apps/condo/domains/billing/components/BillingPageContent/ReceiptsTable.tsx

```
@@ -1,33 +1,42 @@
+    ActionBarProps,
+    Button,
+    FullTableState,
+    GetTableData,
+    Input,
+    RowSelectionState,
+    Table,
+    TableRef,
+} from '@open-condo/ui'
 
+import { BillingReceiptForOrganization as BillingReceiptForOrganizationGQL } from '@condo/domains/billing/gql'
 import { ServicesModal } from '@condo/domains/billing/components/BillingPageContent/ServicesModal'
```

ESLint

[error] `@condo/domains/billing/components/BillingPageContent/ServicesModal` import should occur before import of `@condo/domains/billing/gql`

(import/order)